test/TestCases/VctrBaseMemberFunctions.cpp

```diff
@@ -352,3 +352,37 @@ TEST_CASE ("shift", "[VctrBaseMemberFunctions]")
         REQUIRE_THAT (zeros, vctr::Equals (a));
     }
 }
+
+TEST_CASE ("fill", "[VctrBaseMemberFunctions]")
+{
+    constexpr int fillValue = 5;
+
+    vctr::Array a { 0, 1, 2, 3, 4 };
+    a.fill (fillValue);
+    REQUIRE (a.sum() == fillValue * a.size());
+
+    vctr::Vector b { 0, 1, 2, 3, 4 };
+    b.fill (fillValue);
+    REQUIRE (b.sum() == fillValue * b.size());
+}
+
+TEST_CASE ("init", "[VctrBaseMemberFunctions]")
+{
+    constexpr int fillValue = 13;
+
+    SECTION ("Reduce size")
+    {
+        vctr::Vector b { 0, 1, 2, 3, 4 };
+        b.init (3, fillValue);
+        REQUIRE (b.size() == 3);
+        REQUIRE (b.sum() == fillValue * b.size());
```

Comment:
I'd rather go for `REQUIRE (b.allElementsEqual (fillValue))`

Reply:
Thx, didn't know about the method.